mod-source-record-manager-server/src/test/java/org/folio/services/JournalUtilTest.java

```diff
@@ -163,6 +163,39 @@ public void shouldBuildJournalRecordWhenNoEntityPresent() throws JournalRecordMa
     Assert.assertNotNull(journalRecord.get(0).getActionDate());
   }
 
+  @Test
+  public void shouldBuildJournalRecordForSingleHolding() throws JournalRecordMapperException {
+    String instanceId = UUID.randomUUID().toString();
+    String holdingsId = UUID.randomUUID().toString();
+    String holdingsHrid = UUID.randomUUID().toString();
+
+    JsonObject holdingsJson = new JsonObject()
+      .put("id", holdingsId)
+      .put("hrid", holdingsHrid)
+      .put("instanceId", instanceId);
+
+    String recordId = UUID.randomUUID().toString();
+    String snapshotId = UUID.randomUUID().toString();
+
+    JsonObject recordJson = new JsonObject()
+      .put("id", recordId)
+      .put("snapshotId", snapshotId)
+      .put("order", 1);
+
+    HashMap<String, String> context = new HashMap<>();
+    context.put(HOLDINGS.value(), holdingsJson.encode());
+    context.put(MARC_BIBLIOGRAPHIC.value(), recordJson.encode());
```

Comment:
propose to use MARC_HOLDINGS instead of MARC_BIBLIOGRAPHIC to describe this case of marc-holdings import more accurately